xml/admin_logging.xml

```diff
@@ -0,0 +1,446 @@
+         <para>
+          kubelet
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          kube proxy
+         </para>
+        </listitem>
+        </itemizedlist>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+  </itemizedlist>
+
+  <sect2 xml:id="sec.admin.logging.log-levels.salt">
+   <title>Salt Log Levels</title>
+
+   <para>
+    Salt provides different loglevels that apply both to the master and the
+    minions.
+   </para>
+
+   <itemizedlist>
+    <listitem>
+     <para>
+      all
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      garbage
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      trace
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      debug
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      profile
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      info
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      warning
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      error
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      critical
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      quiet
+     </para>
+    </listitem>
+   </itemizedlist>
+
+   <para>
+   For detailed explanations of the usage of these log levels please see:
+   <link xlink:href="https://docs.saltstack.com/en/latest/ref/configuration/logging/">Salt Log Levels (Upstream)</link>
+   </para>
+
+   <sect3 xml:id="sec.admin.logging.log-levels.salt.debug">
+    <title>Setting a different log level</title>
+    <para>
+    When debugging issues, <literal>debug</literal> is usually enough and is
+    what the &productname; engineers use to debug problems.
+    </para>
+
+    <sect4 xml:id="sec.admin.logging.log-levels.salt.debug.master">
+     <title>Salt Master</title>
+
+    <para>
+     In the case of the <literal>salt-master</literal>, this configuration can be
+     modified in the admin node, at <filename>/etc/caasp/salt-master-custom.conf</filename>.
+     Inside this file you can add: <literal>log_level: debug</literal>.
+    </para>
+
+    <para>
+     Note that after any change on this file you need to restart the
+     <literal>salt-master</literal> container, like:
+    </para>
+<screen>
+<command>docker rm -f $(docker ps | grep salt-master | awk '{print $1}')</command>.
+     </screen>
+
+    <para>
+     After deleting this container, the <literal>kubelet</literal> will bring a
+     new <literal>salt-master</literal> container automatically with the new
+     configuration applied. So you can check the logs with the
+     <literal>debug</literal> loglevel.
+     </para>
+
+<screen>
+<command>docker logs -f $(docker ps | grep salt-master | awk '{print $1}')</command>
+   </screen>
+  </sect4>
+
+  <sect4 xml:id="sec.admin.logging.log-levels.salt.debug.minion">
+   <title>Salt Minions</title>
+   <note>
+    <title>Modifications on Salt minions are not persistent</title>
```

Comment:
I don't think this is very accurate; AFAIK if you create as mentioned a new file `/etc/salt/minion.d/100-debug.conf`, this will be kept even after a new snapshot is created by `transactional-update`.

Reply:
What I think makes sense to explain is that **any** manual modification to files handled by `salt` (that is: managed by the `salt-minions` in the end) will be lost after salt executes any state that touches those files again; however I don't think this is the case for `/etc/salt/minion.d`.

Reply:
I can confirm that I created `/etc/salt/minion.d/100-debug.conf` in a 3.0 installation (2016.11.4). Added a repo that upgraded salt to 2018.3.0 to test, and after `transactional-update pkg install salt-2018.3.0-1.4.x86_64` and rebooting the machine, salt got upgraded and I could still see the `/etc/salt/minion.d/100-debug.conf`.

I don't know if this can be encouraged or it could bring some issues in the future that we should clearly explain (@thkukuk can help here).